__tests__/app.tests.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-env jest */
+const request = require('request')
+const configPaths = require('../config/paths.json')
+const PORT = configPaths.port
+
+let browser
+let page
+
+const requestParamsHomepage = {
+  url: `http://localhost:${PORT}/`,
+  headers: {
+    'Content-Type': 'text/plain'
+  }
+}
+
+beforeAll(async (done) => {
+  browser = global.browser
+  page = await browser.newPage()
+  done()
+})
+
+afterAll(async (done) => {
+  await page.close()
+  done()
+})
+
+describe('Homepage', () => {
+  it('should resolve with a http status code of 200', done => {
```

Comment:
Is this really just testing BrowserSync? The only case I can think of where this would fail is If the metalsmith build failed, which should cause the build to fail anyway.